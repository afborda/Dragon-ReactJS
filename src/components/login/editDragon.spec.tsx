import { render } from '@testing-library/react'
import React from 'react'
import Login from '.'

describe('<ListDragon/>', () => {
  it('should  render the heading', () => {
    const { container } = render(<Login />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
