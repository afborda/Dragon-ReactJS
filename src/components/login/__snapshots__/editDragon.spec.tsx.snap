// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListDragon/> should  render the heading 1`] = `
<div
  class="style__CustomDiv-btqmCT jaKSvJ"
>
  <div
    class="style__Center-jZjGrQ ieRdVQ"
  >
    <div>
      <div>
        <div>
          <img
            alt="Image de perfil do usuario"
            class="style__CustomImg-gVgzYn iFHTNp"
            src="test-file-stub"
          />
        </div>
      </div>
      <form>
        <div>
          <input
            class="style__InputType-eWYwgd bgsYnK"
            placeholder="Digite seu e-mail"
            required=""
            type="email"
          />
          <input
            class="style__InputType-eWYwgd bgsYnK"
            placeholder="Digite sua senha"
            required=""
            type="password"
          />
        </div>
        <div>
          <button
            class="style__Buttom-gGydxe kWrCVF"
            type="submit"
          >
            Entrar
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`
